winml/test/api/LearningModelAPITest.cpp

```diff
@@ -227,6 +234,7 @@ TEST_F(LearningModelAPITest, CloseModelCheckMetadata)
 
 TEST_F(LearningModelAPITestGpu, CloseModelCheckEval)
 {
+    GPUTEST
```

Comment:
>GPUTEST [](start = 4, length = 7)

shouldn't need this, right?

Reply:
We need this for now, it's in the PR description 😄 :

* Add GPUTEST / SKIP_EDGECORE macros
  * Currently, they are added to each test instead of only to the fixture. The googletest submodule version is outdated and doesn't support GTEST_SKIP, which would let us skip tests in the fixture setup and avoid duplicating the GPUTEST macro everywhere. GPUTEST can be removed from each test once we update googletest

Reply:
oh ok. my bad.

---
In reply to: [350506373](https://github.com/microsoft/onnxruntime/pull/2479#discussion_r350506373) [](ancestors = 350506373)